src/Pages/SignUp.tsx

```diff
@@ -9,35 +9,34 @@ import { useFormik } from "formik";
 
 const validationSchema = yup.object({
   email: yup.string().required("Please enter your email address"),
+  displayName: yup.string().required("Display name is required"),
   password: yup.string().required("Please enter a password"),
   confirmPassword: yup.string().required("Please confirm your password"),
 });
 
 function SignUpPage() {
-  const { signup, setRegisterEmail, setRegisterPassword, googleSignIn } =
-    useAuth();
-  const emailRef = useRef<null | HTMLInputElement>(null);
-  const passwordRef = useRef<null | HTMLInputElement>(null);
-  const passwordConfirmationRef = useRef<null | HTMLInputElement>(null);
-  const navigate = useNavigate();
+  const { signup, googleSignIn } = useAuth();
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+  const [confirmationPassword, setConfirmationPassword] = useState<string>("");
+  const [displayName, setDisplayedName] = useState<string>("");
 
-  const handleSignup = async () => {
-    if (passwordRef.current?.value !== passwordConfirmationRef.current?.value) {
-      return console.log("Passwords do not match");
+  const handleSubmit = async () => {
```

Comment:
Can you please rename this to handleSignUp as it was in main? :)

Reply:
yez